pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -1122,6 +1122,9 @@ func appNetworkDoActivateUnderlayNetwork(
 	ulStatus.Vif = vifName
 	ulStatus.Mac = appMac
 	ulStatus.HostName = config.Key()
+	for _, ifName := range netInstStatus.IfNameList {
+		ulStatus.Uplinks = append(ulStatus.Uplinks, "k"+ifName)
```

Comment:
Why do we care about tracking the k-names here? I think everywhere else the uplinks use the names without the k and for e.g., a local network instance over wwan0 there isn't a kwwan0.

In the places where we need to use the k-name (such as retrieving hardware statistics) we have some function to find+check that name's existence so I think it is best to record the ifNames here and find+check where used.

Reply:
Thanks